features/FEATURE_BLE/ble/Gap.h

```diff
@@ -667,6 +684,20 @@ class Gap {
          */
         const ConnectionParams_t   *connectionParams;
 
+        /**
+         * Resolvable address used by the peer.
+         *
+         * @note All bytes of the address are set to 0 if not applicable
```

Comment:
isn't invalid address defined somwhere else as FFFFFFFF?

Reply:
Just saw that:  `ble::addres_t` defines it as FF:FF:FF:FF:FF:FF while the default address in `BLEProtocol::Address_t` is 00:00:00:00:00. Technically both are correct as an address is considered invalid if all its bits are identical. All 0 or all 1 in this case
In the controller specification, when the enhanced connection event is received, resolvable addresses not present are set to 00:00:00:00:00.  Would it be possible to unify our code by setting invalid address in `ble::addres_t` as all 0's ?

Reply:
good idea